Node/src/TeamsChatConnector.ts

```diff
@@ -161,6 +166,45 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  *  Return a newly started reply chain address in channel
+  *  @param {string} serverUrl - Server url is composed of baseUrl and cloud name, remember to find your correct cloud name in session or the function will not find the team.
+  *  @param {string} channelId - The channel id, will post in the channel.  
+  *  @param {builder.Message} message - The message to post in the channel.
+  *  @param {function} callback - This callback returns err or result.
+  */
+  public beginReplyChainInChannel(serverUrl: string, channelId: string, message: builder.Message, callback: (err: Error, address: builder.IChatConnectorAddress) => void) : void {
```

Comment:
Why "beginReplyChainInChannel", and not "startReplyChain", to be more similar to "startConversation"?
The "InChannel" part seems redundant since reply chains are always in channels.